spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/headers.adoc

```diff
@@ -179,3 +179,61 @@ MessagingMessageConverter converter() {
 
 If using Spring Boot, it will auto configure this converter bean into the auto-configured `KafkaTemplate`; otherwise you should add this converter to the template.
 
+[[multi-value-header]]
+== Support multi-value header
+
+Spring for Apache Kafka 4.0 supports multi-value header where same logical header key appears more than once in a Kafka record.
+
+By default, when multiple headers share the same name, the `HeaderMapper` treats them as a single value and serialises the collection to JSON.
+
+* **Producer side:** `DefaultKafkaHeaderMapper` writes the JSON bytes, while `SimpleKafkaHeaderMapper` ignore it.
```

Comment:
I wonder whether we should show an example of how a producer application adds multi headers. For ex. when reading the docs, it might be beneficial, if I can see how multi headers are produced. What do you think?